platform/podjs/src/browserPod.ts

```diff
@@ -307,15 +307,32 @@ class IDBPolyOut implements PolyOut {
     }
 
     async stat(id: string): Promise<Stats> {
-        if (id != "") return (await this.getFile(id)).stat();
-        return {
-            getId: () => "",
-            getSize: () => 0,
-            getTime: () => "",
-            getName: () => "",
-            isFile: () => false,
-            isDirectory: () => true,
-        };
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const stats: any =
+            id != ""
+                ? await (await this.getFile(id)).stat()
+                : {
+                      getId: () => "",
+                      getSize: () => 0,
+                      getTime: () => "",
+                      getName: () => "",
+                      isFile: () => false,
+                      isDirectory: () => true,
+                  };
+
+        // While these properties aren't part of the interface, they are
+        // incidentally present in other PolyPod implementations and features
+        // rely on them, so we return them in addition to the accessors defined
+        // in the interface. We should get rid of either the accessors or the
+        // value properties globally though.
+        stats.id = stats.getId();
+        stats.size = stats.getSize();
+        stats.time = stats.getTime();
+        stats.name = stats.getName();
+        stats.file = stats.isFile();
+        stats.directory = stats.isDirectory();
```

Comment:
Does the code uses both `file` and `directory`? Seems kinda redundant

Reply:
> > I don't know what kind of rats tail that's gonna get me - for a quick fix this already took the entire day :D I'd rather merge a fix first and worry about changing things in _pod-api_ later.
> 
> I merely suggested to mark the methods as deprecated for now, not removing them just yet. What could possibly go wrong? But I guess, you don't have to.

I'd also have to add the properties to the interface then, and who knows what other yak shaving... I opted for a localised fix to get this done faster.

Reply:
> Does the code uses both `file` and `directory`? Seems kinda redundant

Yeah, that's how it was for some reason, thought it was Node.js compatibility. I made `CompatStats` a bit smarter that way, FWIW.